### ISSUE TITLE
livereload.js does not load when using an alternative base path  

### ISSUE THREAD

**Issue by witchof0x20** [Keats/gutenberg#2742]
# Bug Report
I am locally testing a zola project using
`zola serve --base-url http://localhost/project`

This is to mimic how github pages lays out a subdirectory, so I can make sure my links are robust to being on a subdirectory.

However, when serving, zola injects `<script src="/livereload.js?port=1024&amp;mindelay=10"></script></body>`

because this isn't under the base path, this script fails to load.

I think the fix is just adding the base url to the format string here: https://github.com/getzola/zola/blob/master/components/site/src/lib.rs#L576

## Environment

Zola version: 0.19.2

## Expected Behavior
livereload.js loads properly

## Current Behavior
GET http://localhost:1111/livereload.js 404s

## Step to reproduce
`zola serve --base-url http://localhost/project`

**Comment by harrymkt** [comment]
```
<script src="{{ get_url(path = "livereload.js") }}?port=1024&amp;mindelay=10"></script></body>
```

**Comment by witchof0x20** [comment]
livereload.js isn't something I'm trying to import myself, it's a [component of zola serve ](https://github.com/getzola/zola/blob/master/src/cmd/serve.rs#L68)

**Comment by witchof0x20** [comment]
I could probably hack that in, but it seems like a pretty small codechange to fix it in Zola itself.

**Comment by Keats** [comment]
I think that's fine to do a PR to add that to the next branch

**Comment by harrymkt** [comment]
> @witchof0x20 I could probably hack that in, but it seems like a pretty small codechange to fix it in Zola itself.

No, because some may want to reference the functions from the main website, for example, say:

web is the main website, and subweb is the subfolder in it, like GitHub pages.

And then user places the functions.js in the main website (`web`), and reference in the `subweb` using `/functions.js`, thereby the JS is fetched from the main repository, no need to copy over.

Of course if this option would be implemented, a config option to turn this off will need to be advised alongside that one.

**Comment by karamfd** [comment]
The `zola serve` command stopped working for me as well when making changes to static pages locally. 

The pages do not automatically update and I have to restart the server each time I make a new change. 

I'm using zola 0.19.2 on Mac OS Sequoia 15.2

**Comment by Keats** [comment]
livereload.js is now served by the zola server as a route directly so it should work.
